definitions/infra-host/golden_metrics.yml

```diff
@@ -74,8 +74,7 @@ networkTrafficTx:
       eventId: entityGuid
       eventName: entityName
     opentelemetry:
-      # emulate a rate of 1 second in order to get bytes/second
-      select: sum(system.network.io) / sum((endTimestamp - timestamp) / 1000)
+      select: rate(sum(system.network.io), 1 second)
```

Comment:
if you're going to change this one, please also change the networkTrafficRx metric